common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -144,6 +144,24 @@ public static AutoProvMode fromString(String s) throws ServiceException {
         public boolean isMANUAL() { return this == MANUAL;}
     }
 
+    public static enum BackupBlobsCompressType {
+        zip("zip"),
+        zipStore("zipStore"),
+        noZip("noZip");
+        private String mValue;
+        private BackupBlobsCompressType(String value) { mValue = value; }
+        public String toString() { return mValue; }
+        public static BackupBlobsCompressType fromString(String s) throws ServiceException {
+            for (BackupBlobsCompressType value : values()) {
+                if (value.mValue.equals(s)) return value;
+             }
+             throw ServiceException.INVALID_REQUEST("invalid value: "+s+", valid values: "+ Arrays.asList(values()), null);
+        }
```

Comment:
Looks like you're just following the pattern of other enums here, but there's already a `toString` and `valueOf` on `Enum` that accomplish (almost) the same behavior.

No action necessary. Just sayin'.